test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -497,6 +511,115 @@ HapiApiSpec ETX_009_callsToTokenAddresses() {
+		final var contract = "TokenCreateContract";
+		final var EXISTING_TOKEN = "EXISTING_TOKEN";
+		final var firstTxn = "firstCreateTxn";
+		final long DEFAULT_AMOUNT_TO_SEND = 20 * ONE_HBAR;
+
+		return defaultHapiSpec("fungibleTokenCreateWithFeesHappyPath")
+				.given(
+						newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+						cryptoCreate(RELAYER).balance(6 * ONE_MILLION_HBARS),
+						cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS))
+								.via("autoAccount"),
+
+						cryptoCreate(feeCollector)
+								.balance(ONE_HUNDRED_HBARS),
+						uploadInitCode(contract),
+						contractCreate(contract)
+								.gas(GAS_LIMIT),
+						tokenCreate(EXISTING_TOKEN)
+								.decimals(5),
+						tokenAssociate(feeCollector, EXISTING_TOKEN)
+				).when(
+						withOpContext(
+								(spec, opLog) ->
+										allRunFor(
+												spec,
+												ethereumCall(contract,
+														"createTokenWithAllCustomFeesAvailable",
+														spec.registry().getKey(
+																SECP_256K1_SOURCE_KEY).getECDSASecp256K1().toByteArray(),
+														asAddress(spec.registry().getAccountID(feeCollector)),
+														asAddress(spec.registry().getTokenID(EXISTING_TOKEN)),
+														asAddress(spec.registry().getAccountID(RELAYER)),
+														8_000_000L)
+														.via(firstTxn)
+														.gasLimit(GAS_LIMIT)
+														.sending(DEFAULT_AMOUNT_TO_SEND)
+
+														.exposingResultTo(result -> {
+															log.info("Explicit create result is {}", result[0]);
+															final var res = (byte[]) result[0];
+															createdTokenNum.set(new BigInteger(res).longValueExact());
+														})
+
+						))
+				).then(
+						getTxnRecord(firstTxn).andAllChildRecords().logged(),
+						getAccountBalance(RELAYER).logged(),
+						getAccountBalance(contract).logged(),
+						getContractInfo(contract).logged(),
+						childRecordsCheck(firstTxn, ResponseCodeEnum.SUCCESS,
+								TransactionRecordAsserts.recordWith()
+										.status(ResponseCodeEnum.SUCCESS)),
+						sourcing(() -> {
+							final var newToken = asTokenString(TokenID.newBuilder()
+									.setTokenNum(createdTokenNum.get())
+									.build()
+							);
+							return getTokenInfo(newToken).logged()
+									.hasTokenType(TokenType.FUNGIBLE_COMMON)
+									.hasSymbol("tokenSymbol")
+									.hasName("tokenName")
+									.hasDecimals(8)
+									.hasTotalSupply(200)
+									.hasEntityMemo("memo")
+									.hasTreasury(contract)
+									.hasSupplyType(TokenSupplyType.INFINITE)
+									.searchKeysGlobally()
+									.hasCustom(fixedHtsFeeInSchedule(1, EXISTING_TOKEN, feeCollector))
+									.hasCustom(fixedHbarFeeInSchedule(2, feeCollector))
+									.hasCustom(fixedHtsFeeInSchedule(4, newToken, feeCollector))
+									.hasCustom(
+											fractionalFeeInSchedule(4, 5, 10, OptionalLong.of(30), true, feeCollector));
+						})
+				);
+	}
```

Comment:
Perhaps you can add one more E2E test for 007 test case, where you inspect the `transactionRecord` returned directly from the `ethereumCall`, e.g.:

```
final var record = op.getResponseRecord();
final var callResult = record.getContractCallResult();
//inspect callResult
```